### DISCUSSION TITLE
500: INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by iamraphealc** [vercel/community#2799] [Help]
### Summary

FUNCTION_INVOCATION_FAILED

### Steps to Reproduce

I get This Serverless Function has crashed. when trying to connect to my api, but this does not happen locally

**Comment by smaeda-ks** [comment]
same as https://github.com/orgs/vercel/discussions/2800, closing

**Reply by iamraphealc** [reply]
No it's still same issue
